.github/workflows/assets.yml

```diff
@@ -75,6 +75,15 @@ jobs:
              mv assets/kernel assets/kernel.gz
              gzip -d assets/kernel.gz
           fi
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build SBOM and collected sources
+        run: |
+          rm -rf dist
```

Comment:
Nothing else in the workflow needs to remove dist. Why do you need it here?

Reply:
we added the dist earlier just to be on safe side as our initial plan was to use find. Now we are using the actual output from the make command so we don't need it.